data/jyotir07.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Jyotiraditya Singh",
+  "type": "personal",
+  "bio": "**Your Bio**",
+
+  "tags": ["WebDev", "tag2", "tag3"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
+    { "icon": "FaGithub", "url": "https://github.com/<Your GitHub Username>" }, //for mini icons
+    {
+      "icon": "FaLinkedin",
+      "url": "Your Linkedin Profile URL"
+    }
+  ],
+
+  "links": [
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/jyotir07",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/<Your Twitter Username>",
+      "icon": "FaTwitter"
```

Add your Twitter GitHub usernames, if not, remove the section.

```suggestion
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/<Your Twitter Username>",
      "icon": "FaTwitter"
```